tests/run/cpdef_enums.pyx

```diff
@@ -131,3 +131,15 @@ def check_docs():
     'Home is where...'
     """
     pass
+
+
+def to_from_py_conversion(PxdEnum val):
+    """
+    >>> to_from_py_conversion(RANK_1) is PxdEnum.RANK_1
+    True
+    >>> to_from_py_conversion(500)
+    Traceback (most recent call last):
+    ...
+    ValueError: 500 is not a valid PxdEnum
```

Comment:
What about an existing integer value? Does that work?

Reply:
Not sure what you mean by "existing integer value"?